aws/eks/locals.tf

```diff
@@ -9,18 +9,22 @@ locals {
     var.tags,
   )
 
+
   cluster_features_defaults = {
     "cluster_autoscaler" = true
     "metrics_server"     = true
     "velero"             = true
     "external_secrets"   = true
+    "configmapsecrets"   = true
```

Comment:
Do we want this to default to true? Seems like this would affect all existing users of this module.